front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,206 @@
+import React, { useEffect, useState, useRef } from 'react'
+import Header from './Header'
+import Footer from './Footer'
+import './IndividualShow.css';
+import axios from 'axios';
+import { createMockUser, mockShowAPI, mockUserImage, mockShowImage } from './MockData'
+
+
+/*the component stores user's watched episode progress 
+by allowing to save the show's latest season and episode watched*/
+const ProgressData = ({ season, episode, isMovieSet }) => {
+  const refSeason = useRef();
+  const refEpisode = useRef();
+  const saveProgressData = (() => {
+    //the function will be further developed in sprint 2
+    let progress = [];
+    let seasonS = React.findDOMNode(this.refs.season).value;
+    let episodeS = React.findDOMNode(this.refs.episode).value;
+    progress.push(seasonS);
+    progress.push(episodeS);
+  }); //return null if the show is a movie since movie has typically only one season/episode
+  if (isMovieSet==true) {
```

Comment:
This should probably be ===, I think

Reply:
not sure hehe, it shows an error message, maybe you could test it if want?

Reply:
Switching to === suppresses the warning and runs normally, so we should probably use that